shared/featureconfig/config_test.go

```diff
@@ -10,23 +10,23 @@ import (
 
 func TestInitFeatureConfig(t *testing.T) {
 	cfg := &Flags{
-		SkipBLSVerify: true,
+		MedallaTestnet: true,
 	}
 	Init(cfg)
 	c := Get()
-	assert.Equal(t, true, c.SkipBLSVerify)
+	assert.Equal(t, true, c.MedallaTestnet)
 
 	// Reset back to false for the follow up tests.
-	cfg = &Flags{SkipBLSVerify: false}
+	cfg = &Flags{MedallaTestnet: false}
```

Comment:
can we use some other config value ? `MedallaTestnet` is currently enabled by default at the moment

Reply:
Does it matter for a test though? I mainly wanted something that stays, it's unlikely we will delete a popular testname flag

Reply:
@nisdas it doesn't matter in this context. Init basically just loads whatever the flags are. 
```go
// Init sets the global config equal to the config that is passed in.
func Init(c *Flags) {
	featureConfigLock.Lock()
	defer featureConfigLock.Unlock()

	featureConfig = c
}
```